torch/_dynamo/variables/builder.py

```diff
@@ -2893,23 +2918,6 @@ def is_dynamic_source(source_name: str) -> bool:
     return False
 
 
-# Tracks the sources of all fake tensors we wrap in Dynamo.
-# Used by shape guard computation.
-@dataclasses.dataclass
-class TrackedFake:
```

Comment:
moved this to symbolic_shapes.py to prevent circular imports since export uses this directly